estate/models/estate_property.py

```diff
@@ -0,0 +1,24 @@
+from odoo import models, fields
+from dateutil.relativedelta import relativedelta
+
+class EstateProperty(models.Model):
+    _name = 'estate.property'
+    _description = "Properties in sale."
+
+    name = fields.Char(required=True)
+    description = fields.Char(help=None)
+    postcode = fields.Char(help=None)
+    date_availability = fields.Date(default=lambda self: fields.Datetime.now() + relativedelta(months=3), copy=False)
```

Comment:
the lambda function to fill the field isn't needed as you don't use the environment. A possible use of the lambda function would have been: default=; lambda self: self._set_default_date_availability() with _set_default_date_availibilty() referring to a method like
    def _default_date_availability(self): return fields.Date.context_today(self) + relativedelta(months=3)

Here as it's kinda straight forward, you may directly put in the declaration of the field :
default=fields.Date.today() + relativedelta(months=3)